src/protocol/tx-validity.md

```diff
@@ -206,11 +215,25 @@ Transaction processing is completed by removing spent UTXOs from the state and a
 
 ### Coinbase Transaction
 
-The coinbase transaction is a mechanism for block creators to convert fees into spendable UTXOs.
+The coinbase transaction is a mechanism for block creators to collect transaction fees.
 
 In order for a coinbase transaction to be valid:
 
 1. It must be a [Mint](../tx-format/transaction.md#TransactionMint) transaction.
-2. The coinbase transaction must be the first transaction within a block, even if there are no other transactions in the block and the fee is zero.
-3. The total output value of the coinbase transaction cannot exceed the total amount of fees processed from all other transactions within the same block.
-4. The `asset_id` for coinbase transaction outputs must match the `asset_id` that fees are paid in (`asset_id == 0`).
+1. The coinbase transaction must be the first transaction within a block, even if there are no other transactions in the block and the fee is zero.
```

Comment:
It should be the last transaction in the block to avoid affecting other transactions.

Reply:
Fine with me

Reply:
updated